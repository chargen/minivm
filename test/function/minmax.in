print min(10, 20)
print min(20, 10)
print max(10, 20)
print max(20, 10)

a = max(min(20, 40), max(30, 10))
print a

print max(3, 1, 4, 5, 2)
print min(3, 1, 4, 5, 2)

print min(3.5, 6, 1.3)
print max(7.3, 9, 4.0)
