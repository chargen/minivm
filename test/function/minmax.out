10
10
20
20
30
5
1
1.300000000
9.000000000
